Auth0/SafariWebAuth.swift

```diff
@@ -100,6 +101,11 @@ class SafariWebAuth: WebAuth {
         return self
     }
 
+    func useLegacyAuthentication() -> Self {
```

Comment:
Not a big fun but can't think of a better name 😄

Reply:
Naming is the hardest part of development 😆